CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ option(BUILD_SAMPLE "Build available samples" ON)
 option(ENABLE_DATA_CHANNEL "Enable support for data channel" ON)
 option(INSTRUMENTED_ALLOCATORS "Enable memory instrumentation" OFF)
 option(ENABLE_AWS_SDK_IN_TESTS "Enable support for compiling AWS SDKs for tests" ON)
+option(USE_VIDEO_H265 "Use h265 video frames " ON)
+option(USE_AUDIO_AAC "Use aac audio frames " ON)
```

Comment:
What is the reason we are using cmake flags to choose codecs here when we're not using the same approach for the existing codecs?